RELEASE.rst

```diff
@@ -1,7 +1,3 @@
 RELEASE_TYPE: patch
 
-This is a refactoring release. It moves a number of internal uses
-of nametuple over to using attrs based classes, and removes a couple
-of internal namedtuple classes that were no longer in use.
-
-It should have no user visible impact.
+This is a patch release that fixes a bug in the `hypothesis.extra.pandas` documentation I changed column to columns. 
```

Comment:
This is going to go into the changelog for the release, so the style should match that. I'd probably just replace " I changed column to column" to "where it incorrectly referred to column instead of columns".